stacks-signer/src/runloop.rs

```diff
@@ -190,14 +190,11 @@ impl<C: Coordinator> RunLoop<C> {
             .signing_round
             .process_inbound_messages(&inbound_messages)
             .unwrap_or_default();
-        // If the signer is the coordinator, then next process the message as the coordinator
-        let (messages, results) = if self.signing_round.signer_id == coordinator_id {
-            self.coordinator
-                .process_inbound_messages(&inbound_messages)
-                .unwrap_or_default()
-        } else {
-            (vec![], vec![])
-        };
+        // Next process the message as the coordinator
+        let (messages, results) = self
+            .coordinator
+            .process_inbound_messages(&inbound_messages)
+            .unwrap_or_default();
```

Comment:
What does `unwrap_or_default()` do here? Does this mean if there's an error processing the inbound messages, the runloop just ignores the inbound messages? Should this be logged?

Reply:
It will ignore the error and return an empty vector. In general we never want the coordinator failing to process a message to crash and should continue processing the next event. We could later try to special case handle errors or I could add a print statement, but so far there has been no desire/need. If you want it logged, I would open another PR for this just to get this through. XD

Reply:
Yeah -- I think there's logging in most places that would yield an error here, but I think it'd be good to add a catch all log there.

Reply:
Done :)